src/stories/Toggle.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { Toggle } from '..';
+
+export default {
+  title: 'Form/Toggle',
+  component: Toggle,
+} as Meta<typeof Toggle>;
```

Comment:
if [this](https://github.com/ubie-oss/ubie-ui/pull/19) approved

```suggestion
} satisfies Meta<typeof Toggle>;
```